CHANGELOG.rst

```diff
@@ -1,5 +1,29 @@
 Changelog
 =========
+1.4.0 (2019-10-30)
+-----------------------
+MLflow 1.4.0 includes several major features:
+
+- Model Registry. MLflow 1.4 adds an experimental model registry feature, where you can manage, version, and keep lineage of your production models. (#1943, @mparkhe, @Zangr, @sueann, @dbczumar, @smurching, @gioa, @clemens-db, @pogil, @mateiz)
+- Keras support for TensorFlow 2.0. MLflow Keras model saving, loading, and logging has been updated to be compatible with TensorFlow 2.0.  (#1927, @juntai-zheng)
+- Autologging for TensorFlow 2.0. Autologging for ``tf.estimator`` and ``tf.keras`` models has been updated to be compatible with TensorFlow 2.0. The same functionalities of autologging in TF 1.x will be offered in TF 2.0, namely when fitting tf.keras models and when exporting saved tf.estimator models. (#1910, @juntai-zheng)
+
+More features and improvements:
+
+- Add functions ``get_run``, ``get_experiment``, ``get_experiment_by_name`` to the fluent API (#1923, @fhoering)
+- Use Plotly as artifact image viewer to make it interactive (#1934, @harupy)
```

Comment:
Use Plotly as artifact image viewer to make it interactive 

What does "it" appy to?